homeassistant/components/ring/__init__.py

```diff
@@ -235,7 +237,7 @@ def __init__(
         self.ring = ring
         self.update_method = update_method
         self.update_interval = update_interval
-        self.devices = {}
+        self.devices: dict = {}
```

Comment:
@balloob, as `ring` codeowner could you confirm if there is a better hint we could use here for `self.devices`?